rero/rero-ils
# Patron: Add communication channel

* NEW: Add communication channel and communication language

How to test:
./scripts/bootstrap
./scripts/setup
Add a new patron user and check fields (communication)

Signed-off-by: Bertrand Zuchuat <bertrand.zuchuat@rero.ch>
